packages/eslint-plugin-pf-codemods/lib/rules/v5/dualListSelector-update-onAvailableOptionsSearchInputChanged.js

```diff
@@ -0,0 +1,7 @@
+const { addCallbackParam } = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8793
+module.exports = {
+  meta: { fixable: "code" },
+  create: addCallbackParam(["DualListSelector"], { onAvailableOptionsSearchInputChanged: { defaultParamName: "_event", previousParamIndex: 1, otherMatchers: /^_?(ev\w*|e$)/ } }),
```

Comment:
```suggestion
  create: addCallbackParam(["DualListSelector"], {
    onAvailableOptionsSearchInputChanged: {
      defaultParamName: "_event",
      previousParamIndex: 1,
      otherMatchers: /^_?(ev\w*|e$)/,
    },
    onChosenOptionsSearchInputChanged: {
      defaultParamName: "_event",
      previousParamIndex: 1,
      otherMatchers: /^_?(ev\w*|e$)/,
    },
    onListChange: "_event",
  }),
```

Then we can remove the other 2 rule files as well as 2 of the tester files. We should also update the file names to something like `dualListSelector-update-paramCallbacks`